es-app/src/views/gamelist/DetailedGameListView.cpp

```diff
@@ -24,6 +26,22 @@ DetailedGameListView::DetailedGameListView(Window* window, FileData* root) :
 	mList.setAlignment(TextListComponent<FileData*>::ALIGN_LEFT);
 	mList.setCursorChangedCallback([&](const CursorState& /*state*/) { updateInfoPanel(); });
 
+	// Thumbnail
+	mThumbnail.setOrigin(0.5f, 0.5f);
+	mThumbnail.setPosition(2.0f 2.0f);
+	mThumbnail.setVisible(false);
+	mThumbnail.setMaxSize(mSize.x() * (0.25f - 2*padding), mSize.y() * 0.10f);
+	mThumbnail.setDefaultZIndex(25);
+	addChild(&mThumbnail);
+
+	// Marquee
+	mMarquee.setOrigin(0.5f, 0.5f);
+	// Default to off the screen
+	mMarquee.setPosition(2.0f, 2.0f);
```

Comment:
setVisible should be set for `mMarquee` as well.

Reply:
also Solved